src/test/java/com/ericsson/ei/subscription/SubscriptionHandlerTest.java

```diff
@@ -116,13 +116,13 @@
     private MockMvc mockMvc;
 
     @MockBean
-    private SendHttpRequest springRestTemplate;
+    private HttpRequestSender springRestTemplate;
 
     private static String subRepeatFlagDataBaseName = "eiffel_intelligence";
     private static String subRepeatFlagCollectionName = "subscription_repeat_handler";
 
     @Autowired
-    private SendMail sendMail;
+    private EmailSender mailSender;
```

Comment:
EmailSender === mailSender? Maybe the name of one is wrong.

Reply:
Will fix this